tests/rptest/tests/rpk_cluster_test.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.utils.util import wait_until
+
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.rpk import RpkTool
+from rptest.services.rpk_consumer import RpkConsumer
+
+import threading
+import random
+
+
+class RpkClusterTest(RedpandaTest):
+    def __init__(self, ctx):
+        super(RpkClusterTest, self).__init__(test_context=ctx)
+        self._ctx = ctx
+        self._rpk = RpkTool(self.redpanda)
+
+    def test_cluster_info(self):
+        def condition():
+            brokers = self._rpk.cluster_info()
+
+            if len(brokers) != len(self.redpanda.nodes):
+                return False
+
+            ok = True
+            for i, b in enumerate(brokers, start=1):
+                ok = ok and \
+                    b.address == f'192.168.52.{b.id}:9092'
```

Comment:
add an interface to the redpanda service to query these addresses from the configuration (or wherever its defined, maybe through environment variables?) because this won't remain true in other deployment scenarios.

Reply:
Done